apollo-router/src/plugins/telemetry/tracing/apollo_telemetry.rs

```diff
@@ -99,6 +104,42 @@ const LABEL: Key = Key::from_static_str("graphql.label");
 const CONDITION: Key = Key::from_static_str("graphql.condition");
 const OPERATION_NAME: Key = Key::from_static_str("graphql.operation.name");
 const OPERATION_TYPE: Key = Key::from_static_str("graphql.operation.type");
+const OPERATION_SUBTYPE: Key = Key::from_static_str("graphql.operation.subtype");
+const EXT_TRACE_ID: Key = Key::from_static_str("trace_id");
+
+/// The set of attributes to include when sending to the Apollo Reports protocol.
+const REPORTS_INCLUDE_ATTRS: [Key; 17] = [
+    APOLLO_PRIVATE_REQUEST,
+    APOLLO_PRIVATE_DURATION_NS_KEY,
+    APOLLO_PRIVATE_SENT_TIME_OFFSET,
+    APOLLO_PRIVATE_GRAPHQL_VARIABLES,
+    APOLLO_PRIVATE_HTTP_REQUEST_HEADERS,
+    APOLLO_PRIVATE_HTTP_RESPONSE_HEADERS,
+    APOLLO_PRIVATE_OPERATION_SIGNATURE,
+    APOLLO_PRIVATE_FTV1,
+    PATH,
+    SUBGRAPH_NAME,
+    CLIENT_NAME_KEY,
+    CLIENT_VERSION_KEY,
+    DEPENDS,
+    LABEL,
+    CONDITION,
+    OPERATION_NAME,
+    OPERATION_TYPE,
+];
```

Comment:
For this make sure to use an HashSet at the end because perfs are better when you're checking if it contains an element. Here is an example [here](https://github.com/apollographql/router/blob/dev/apollo-router/src/plugins/telemetry/formatters/json.rs#L32)

Reply:
Oh I think you already did it. Just make sure it's done that way